ballerina-command/src/main/resources/cli-help/ballerina-dist-pull.help

```diff
@@ -0,0 +1,21 @@
+NAME
+       ballerina-dist-pull - Fetch a given distribution and set it as the active version
+
+SYNOPSIS
+       ballerina dist pull <distribution>
+
+
+DESCRIPTION
+       Pull fetches the specified distribution to your local environment and marks it
+       as the active version.
+
+       The update command always fetches the latest patch version, but pull command gives
+       you the flexibility to bring any distribution version.
```

Comment:
Should we say
```suggestion
       you the flexibility to fetch and use any distribution version.
```

or even just "use" instead of "bring"?

Reply:
+1